go-controller/pkg/ovn/gateway/gateway.go

```diff
@@ -61,8 +61,8 @@ func GetGatewayPhysicalIPs(nbClient libovsdbclient.Client, gatewayRouter string)
 	return nil, fmt.Errorf("no physical IPs found for gateway %s", gatewayRouter)
 }
 
-// CreateDummyGWMacBinding creates a mac binding for a fake next hop when a node has no default gateway
-func CreateDummyGWMacBinding(sbClient libovsdbclient.Client, nodeName string) error {
+// CreateDummyGWMacBindings creates a mac binding for a fake next hop when a node has no default gateway
```

Comment:
nit: since u decided to rename the function to make more accurate then will be good to also update the comment to `creates mac bindings` ?